python/src/lib.rs

```diff
@@ -72,11 +68,14 @@ fn get_draft(draft: Option<u8>) -> PyResult<Draft> {
 #[pyfunction]
 #[text_signature = "(schema, instance, draft=None)"]
 fn is_valid(schema: &PyAny, instance: &PyAny, draft: Option<u8>) -> PyResult<bool> {
-    let draft = get_draft(draft).map(Some)?;
     let schema = ser::to_value(schema)?;
     let instance = ser::to_value(instance)?;
-    let compiled =
-        jsonschema::JSONSchema::compile(&schema, draft).map_err(JSONSchemaError::Compilation)?;
+    let mut config = jsonschema::JSONSchemaCompileConfig::default();
```

Comment:
I think that instead of calling `config.set_draft` it is cleaner to detect draft + create a config object, which is 2 steps when the current implementation is 3 steps - create a config, detect draft, set draft

Reply:
I'm not sure I see your point in here ... can you post a snippet of what you have in mind?

Reply:
I'm not sure I see your point in here ... can you post a snippet of what you have in mind?

Reply:
I had in mind the code it was before, but after the latest changes it is exactly what I meant:

```rust
Ok(JSONSchema {
            schema,
            resolver,
            validators,
            context,
        })
```

Reply:
Oh, it seems I confused this one with the lib itself, but it is actually Python bindings. The idea was to create `JSONSchemaCompileConfig` with draft instead of creating a `mut` one and updating it

Reply:
According to the current `JSONSchemaCompileConfig` implementation we still need two lines to do so ... I'm not going to focus on this for now as on a different open comment we're evaluating the idea of using `::options()`.

Keeping this open to ensure that we don't forget about it